src/Symfony/Component/Form/Resources/translations/validators.sk.xlf

```diff
@@ -14,6 +14,126 @@
                 <target>CSRF token je neplatný. Prosím skúste znovu odoslať formulár.</target>
             </trans-unit>
+            <trans-unit id="99">
+                <source>This value is not a valid HTML5 color.</source>
+                <target>Táto hodnota nie je platná HTML5 farba.</target>
+            </trans-unit>
+            <trans-unit id="100">
+                <source>Please enter a valid birthdate.</source>
+                <target>Prosím zadajte platný dátum narodenia.</target>
+            </trans-unit>
+            <trans-unit id="101">
+                <source>The selected choice is invalid.</source>
+                <target>Vybraná možnosť je neplatná.</target>
+            </trans-unit>
+            <trans-unit id="102">
+                <source>The collection is invalid.</source>
+                <target>Kolekcia je neplatná.</target>
+            </trans-unit>
+            <trans-unit id="103">
+                <source>Please select a valid color.</source>
+                <target>Prosím vyberte platnú farbu.</target>
+            </trans-unit>
+            <trans-unit id="104">
+                <source>Please select a valid country.</source>
+                <target>Prosím vyberte platnú krajinu.</target>
+            </trans-unit>
+            <trans-unit id="105">
+                <source>Please select a valid currency.</source>
+                <target>Prosím vyberte platnú menu.</target>
+            </trans-unit>
+            <trans-unit id="106">
+                <source>Please choose a valid date interval.</source>
+                <target>Prosím vyberte platný rozsah dát.</target>
+            </trans-unit>
+            <trans-unit id="107">
+                <source>Please enter a valid date and time.</source>
+                <target>Prosím zadajte platný dátum a čas.</target>
+            </trans-unit>
+            <trans-unit id="108">
+                <source>Please enter a valid date.</source>
+                <target>Prosím zadajte platný dátum.</target>
+            </trans-unit>
+            <trans-unit id="109">
+                <source>Please select a valid file.</source>
+                <target>Prosím vyberte platný súbor.</target>
+            </trans-unit>
+            <trans-unit id="110">
+                <source>The hidden field is invalid.</source>
+                <target>Skryté pole je neplatné.</target>
+            </trans-unit>
+            <trans-unit id="111">
+                <source>Please enter an integer.</source>
+                <target>Prosím zadajte celé číslo.</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>Prosím vyberte platný jazyk.</target>
+            </trans-unit>
+            <trans-unit id="113">
+                <source>Please select a valid locale.</source>
+                <target>Prosím vyberte platnú lokalizáciu.</target>
```

Comment:
Make sure this refers to a "locale". Example: "en", "fr", "es" "sv". 
It is not a location/place.

Reply:
I changed it to the same translation as `language`, following example of Czech folks here :)

Reply:
I see. 
Is there a word for "locale" though? 

If you would ask me what the correct locale code for Sweden is, what would you say?

Reply:
I would ask `Aký je správny jazykový kód pre Švedsko?` which literally means `What is the correct language code for Sweden?`. I could translate it as `miestne nastavenia` which is `local settings`, but I think it wouldn't make a lot of sense to end user.

Reply:
Is "miestne nastavenia" a very technical (computer related) term? But it is still correct?

If so, I would prefer that. Because "langauge" is not the same thing as "locale". 


So, let's be more correct here than "friendly for the average website visitor"

Reply:
What does @signor-pedro think we should do here?

Reply:
Ufff to be honest, if this error is meant for devs, we all at work just use the english term "locale" and if I read any of the suggestions such as `jazykový kód`, it would take me some time to understand what you mean :D 

But this code is probably also meant for actual users, who I think do not really understand the concept of "locales", I'd keep it as it is now - `Prosím vyberte platný jazyk.`

Reply:
There are many occurrences of `Miestne nastavenia` in GitHub (and Linux-related projects seem to be using that term: https://github.com/Arcolinux-GzR/arco-settings-manager/blob/2efe91e08d115ebb91da1f6f9dd645fe460573cb/src/modules/locale/msm_locale.desktop, https://github.com/KDE/systemsettings/blob/3150b50af1da3d7493154e05b7c1052ed483e472/categories/settings-personalization-regionalsettings.desktop, etc.)

Reply:
It is the most accurate translation indeed, I'd still just keep the `language` the way they have it in Czech.

Businesses who let you chose which language you want to see the website in use the term "language" (and I now it's not the same, because `en-GB` is not a language, but we do not have this kind of a strict division between language and a "language+location" setting)

Reply:
> Businesses who let you chose which language you want to see the website in use the term "language" (and I now it's not the same, because en-GB is not a language, but we do not have this kind of a strict division between language and a "language+location" setting)

I undertand. 

There are two FormTypes: `LanguageType` and `LocaleType`. One is to specify a language and one is for locale. I think most people would use `LanguageType` and if you pick the wrong language it would say "Please select a valid language". 

However, when a developer uses `LocaleType` then I think the default error message should reflect that. (A developer can always change the error message). I think it make more sense to use the most accurate translation in this case, especially since we have the two form types.

Reply:
Ok, I changed it to `miestne nastavenia`.

Reply:
Thank you

Reply:
@Nyholm thank you for explanation, this makes a lot of sense.